feature/user_profile/presentation/src/main/java/com/sanaa/presentation/screen/watchingHistory/WatchingHistoryScreen.kt

```diff
@@ -56,133 +61,161 @@ fun WatchingHistoryScreen(
+                    fadeIn(animationSpec = tween(150, delayMillis = 150))
+                        .togetherWith(fadeOut(animationSpec = tween(150)))
+                },
+                modifier = Modifier.padding(top = 8.dp)
+            ) { selectedMediaType ->
+                when (selectedMediaType) {
+                    MediaTypeUi.MOVIE -> {
+                        GridSection(
+                            genres = state.movieGenres,
+                            mediaList = watchedMovies,
+                            selectedGenreId = state.movieSelectedGenreId,
+                            onGenreClick = interactionListener::onMovieGenreClick,
+                            onMediaClick = { media ->
+                                interactionListener.onMediaClick(media.id, media.mediaTypeUi)
+                            },
+                            onSaveIconClick = interactionListener::onSaveIconClick,
+                            modifier = Modifier.fillMaxSize()
+                        )
+                    }
 
-                MediaTypeUi.TV_SHOW -> {
-                    GridSection(
-                        genres = state.tvShowGenres,
-                        mediaList = watchedTvShows,
-                        selectedGenreId = state.tvShowSelectedGenreId,
-                        onGenreClick = interactionListener::onTvShowGenreClick,
-                        onMediaClick = { media ->
-                            interactionListener.onMediaClick(media.id, media.mediaTypeUi)
-                        },
-                        onSaveIconClick = interactionListener::onSaveIconClick,
-                        modifier = Modifier.fillMaxSize()
-                    )
+                    MediaTypeUi.TV_SHOW -> {
+                        GridSection(
+                            genres = state.tvShowGenres,
+                            mediaList = watchedTvShows,
+                            selectedGenreId = state.tvShowSelectedGenreId,
+                            onGenreClick = interactionListener::onTvShowGenreClick,
+                            onMediaClick = { media ->
+                                interactionListener.onMediaClick(media.id, media.mediaTypeUi)
+                            },
+                            onSaveIconClick = interactionListener::onSaveIconClick,
+                            modifier = Modifier.fillMaxSize()
+                        )
+                    }
                 }
             }
         }
     }
-}
 
+
+
+    state.selectedMediaToSave?.let { mediaItem ->
+        SaveToListBottomSheet(
+            isVisible = state.showSaveToListBottomSheet,
+            mediaId = mediaItem.id.toLong(),
+            onDismiss = interactionListener::onDismissSaveToListBottomSheet,
+            onCreateNewListClick = interactionListener::onCreateNewListClick,
+            onSuccess = {
+                interactionListener.onShowSuccessSnackBar("Add to list succeeded")
```

Comment:
Please use a string resource instead, somthing like:
```suggestion
                interactionListener.onShowSuccessSnackBar(stringResource(R.string.add_to_list_succeeded))
```

Reply:
i had another solution it should depends on string provider only